docs/UserGuide.adoc

```diff
@@ -348,7 +348,8 @@ OR
 
 * it is an existing transaction and has been successfully edited.
 
-WARNING: All rules will be executed in the order they were added.
+WARNING: All rules will be executed from the bottom of the list upwards.
+Typically, this is order in which the rules were added.
```

Comment:
Missed a spot, should be "...this is **the** order in which..."

Reply:
Oops! thanks